hp_list/main.c

```diff
@@ -195,7 +297,7 @@ list_node_t *list_node_new(list_key_t key)
     list_node_t *node = aligned_alloc(128, sizeof(*node));
     assert(node);
     *node = (list_node_t){.magic = LIST_MAGIC, .key = key};
-    (void) atomic_fetch_add(&inserts, 1);
```

Comment:
Once we wrap the `atomic_` functions into `ATOMIC_`, we can do statistics accordingly.